stubs/jsonschema/jsonschema/_utils.pyi

```diff
@@ -13,13 +13,12 @@ class URIDict(MutableMapping[Any, Any]):
 class Unset: ...
 
 def load_schema(name): ...
-def indent(string, times: int = ...): ...
-def format_as_index(indices): ...
-def find_additional_properties(instance, schema) -> None: ...
-def extras_msg(extras): ...
-def types_msg(instance, types): ...
-def flatten(suitable_for_isinstance): ...
-def ensure_list(thing): ...
-def equal(one, two): ...
+def format_as_index(container: str, indices) -> str: ...
+def find_additional_properties(instance: Iterable, schema: Mapping) -> Generator: ...
```

Comment:
Because you're not supposed to use anything from something named `_utils` anyway, it's fine to leave this file as it was. If you want to improve it anyway, you can use `Any` to fix the CI errors in this file. For example:

```suggestion
def find_additional_properties(instance: Iterable[Any], schema: Mapping[Any, Any]) -> Generator[Any, Any, Any]: ...
```

It is also fine to leave `_utils` as it was, and not do any manual changes to it.